TestSuites.py

```diff
@@ -135,3 +135,28 @@ def getAllTestSuite(self):
             return data["error"] , ()
         return '', data["TestSuites"]
 
+    def delTestSuite(self,csv_file, api_key, api_secret):
+        #read csv file and add test suites
+        result_file = ''.join(random.sample(string.ascii_letters + string.digits, 8)) + ".txt"
+        refile = open(result_file, 'a')
+        error_message, testsuites = self.getAllTestSuite()
+        cvs_name_list = []
+        try:
+            with open(csv_file, 'r') as testsuite_csv:
+                content = csv.reader(testsuite_csv, delimiter=',')
+                for row in content:
+                    name = row[0]
+                    cvs_name_list.append(name)
+
+            for x in testsuites:
+                if x["name"] in cvs_name_list:
+                    error_message = self.deleteTestSuite(api_key, api_secret, x["id"])
+                    if error_message != '':
+                        refile.write(error_message + x["name"] + "\n")
```

Comment:
please write the result more clearly. For example:
[SUCCESS]  Delete test suite `name` successfully.
[ERROR] Failed to delete test suite `name`.
`error_message`